guava/src/main/scala/magnolify/guava/semiauto/FunnelDerivation.scala

```diff
@@ -60,7 +60,7 @@ trait FunnelImplicits {
   implicit val intFunnel: Funnel[Int] = Funnels.integerFunnel().asInstanceOf[Funnel[Int]]
   implicit val longFunnel: Funnel[Long] = Funnels.longFunnel().asInstanceOf[Funnel[Long]]
   implicit val bytesFunnel: Funnel[Array[Byte]] = Funnels.byteArrayFunnel()
-  implicit val charSequenceFunnel: Funnel[CharSequence] = Funnels.stringFunnel(Charsets.UTF_8)
+  implicit val charSequenceFunnel: Funnel[CharSequence] = Funnels.unencodedCharsFunnel()
```

Comment:
What do you think about `stringFunnel`? Should we not force an encoding and use `unencodedCharsFunnel` also? We're not funneling it to disk so no encoding is fine?

Reply:
I think Utf8 for String is a reasonable default. Since this is an implicit, the user can choose to override and specify a `Charset` if they choose to for Strings easily.
However for CharSequence we shouldn't enforce an encoding.
The hasher in guava has special handling for utf8 strings (only)